Update devops-command-center.rst (#4129)
mattermost/docs: source/administration/devops-command-center.rst

Adding slash commands to tasks
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Slash commands can be added to tasks to initiate actions as part of your playbook.

Here are some examples:

- Add a communication task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
- One of your tasks may require the channel header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.

Generating test data
^^^^^^^^^^^^^^^^^^^^

> You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insights page.

Author: ```suggestion
You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insight page.
```

- ``/incident test bulk-data``: This command accepts a number of ongoing incidents, a number of ended incidents, a beginning and an end date, and an optional seed. It creates as many ongoing and ended incidents as specified, all of them with their creation date randomly picked between the beginning and end dates. The seed, if available, is used to get reproducible results. The names of the incidents are randomly chosen from a list of incident names and a list of fake company names which are defined in the code. An example command is: ``/incident test create-incident 6utgh6qg7p8ndeef9edc583cpc 2020-11-23 PR-Testing``.

> - ``/incident test create-incident``: This command accepts a playbook ID (that can be chosen from the playbooks the user is a member of, using the autocomplete system), a timestamp, and an incident name. It creates an ongoing incident with the creation date set to the specified timestamp. An example command looks like this: ``/incident test bulk-data 10 3 2020-01-31 2020-11-22 2``.

Reviewer: The examples here are swapped!

Author: Yep, @cwarnermm suggested swapping them which I did but I should have confirmed whether they're in a specific order? If they are, I will swap them around and add a note indicating that one needs to be used prior to the other.

Reviewer 2: Good catch on order of operations for running the commands. In addition, the command examples appear to be for the opposite command where ``/incident test create-incident has an example for bulk-data and vice-versa.

Reviewer: > the command examples appear to be for the opposite command where ``/incident test create-incident has an example for bulk-data and vice-versa

Oh yeah, that's what I meant, sorry for the confusion!

Adding slash commands to tasks
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Slash commands can be added to tasks to initiate actions as part of your playbook.

Here are some examples:

> - Add a task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 

Reviewer 2: Consider clarifying the type of playbook task: "Add a **communication** task called **Sync up** with the slash command..."

> - Add a task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 

Author: ```suggestion
- Add a communication task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
```

> - One of your tasks may require the header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.

Reviewer 2: Is it the _channel_ header that would be changed? If so, consider clarifying as: "One of your tasks may require the channel header to be changed to reflect a new status."

Generating test data
^^^^^^^^^^^^^^^^^^^^

> You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insights page.

Reviewer 2: Is it "incident insight page" or "incident insights page"? Ensure that all references, including Glossary entries below, reference the page consistently.

> - One of your tasks may require the header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.

Author: ```suggestion
- One of your tasks may require the channel header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
```

> - ``/incident test create-incident``: This command accepts a playbook ID (that can be chosen from the playbooks the user is a member of, using the autocomplete system), a timestamp, and an incident name. It creates an ongoing incident with the creation date set to the specified timestamp. An example command looks like this: ``/incident test bulk-data 10 3 2020-01-31 2020-11-22 2``.

Reviewer 2: The example command here likely needs to be switched with the bullet point example below.

> - ``/incident test bulk-data``: This command accepts a number of ongoing incidents, a number of ended incidents, a beginning and an end date, and an optional seed. It creates as many ongoing and ended incidents as specified, all of them with their creation date randomly picked between the beginning and end dates. The seed, if available, is used to get reproducible results. The names of the incidents are randomly chosen from a list of incident names and a list of fake company names which are defined in the code. An example command is: ``/incident test create-incident 6utgh6qg7p8ndeef9edc583cpc 2020-11-23 PR-Testing``.

Reviewer 2: Consider changing "ended incidents" to "closed incidents" to match Glossary entry available later on the page.

Author: Good catch, thanks. I've checked against the slash command and replaced "closed" with "ended" as the slash command we use is ``/incident end``. I've changed this throughout the doc.

> - ``/incident test bulk-data``: This command accepts a number of ongoing incidents, a number of ended incidents, a beginning and an end date, and an optional seed. It creates as many ongoing and ended incidents as specified, all of them with their creation date randomly picked between the beginning and end dates. The seed, if available, is used to get reproducible results. The names of the incidents are randomly chosen from a list of incident names and a list of fake company names which are defined in the code. An example command is: ``/incident test create-incident 6utgh6qg7p8ndeef9edc583cpc 2020-11-23 PR-Testing``.

Reviewer 2: The example command here likely needs to be switched with the bullet point example above for bulk-data

Author: Done, thanks!

Glossary
--------

- **Commander:** The Mattermost user currently responsible for transitioning an incident from ongoing to ended.
- **Incident:** An event requiring the coordinated actions of one or more Mattermost users. An incident is either ongoing or ended.
- **Incident channel:** A Mattermost channel dedicated to real-time conversation about the incident.
- **Incident insight page:** The incident details and analytics page, which also provides the channel export download link. It is not available on mobile.
- **Incident member:** A Mattermost user with access to the corresponding incident channel.
- **Playbook:** A task-based process that's followed in order to resolve an incident.
- **Playbook configuration page:** The playbook configuration and editing page. It is not available on mobile.
- **Stage:** A set of tasks grouped together to achieve a specific goal of the workflow, which generally need to be completed before proceeding to the next stage of the incident resolution process.
- **Tasks:** The individual steps required to complete the stages of an incident. Tasks can optionally be assigned to specific incident participants into stages.

> - **The RHS:** The incident list and incident details displayed on the right hand side of the webapp. It is not available on mobile.

Author: ```suggestion
- **The RHS:** The incident list and incident details displayed on the right hand side (RHS) of the web app. It is not available on mobile.
```

> - **The RHS:** The incident list and incident details displayed on the right hand side of the webapp. It is not available on mobile.

Author: ```suggestion
- **The RHS:** The incident list and incident details displayed on the right hand side (RHS) of the web app. It is not available on mobile.
```

--------------

Please see the `Channel Export plugin documentation <https://mattermost.gitbook.io/channel-export-plugin>`_ for more information.

> Glossary

Reviewer 2: Consider listing glossary entries in alphabetical order for easy scan and retrieval.

--------

- **Incident:** An event requiring the coordinated actions of one or more Mattermost users. An incident is either ongoing or closed.
- **Playbook:** A task-based process that's followed in order to resolve an incident.
- **Tasks:** The individual steps required to complete the stages of an incident. Tasks can optionally be assigned to specific incident participants into stages.
- **Stage:** A set of tasks grouped together to achieve a specific goal of the workflow, which generally need to be completed before proceeding to the next stage of the incident resolution process.
- **Commander:** The Mattermost user currently responsible for transitioning an incident from ongoing to closed.
- **Incident channel:** A Mattermost channel dedicated to real-time conversation about the incident.
- **Incident member:** A Mattermost user with access to the corresponding incident channel.

> - **The RHS:** The incident list and incident details displayed on the right hand side of the webapp. It is not available on mobile.

Reviewer 2: Consider updating to clarify the meaning of RHS and remove potential ambiguity: "The incident list and incident details displayed on the right hand side (RHS) of the web app. The RHS is not available on mobile."
